app/components/README.md

```diff
@@ -0,0 +1,10 @@
+# Components
```

Comment:
This is a good idea!

We already have some content about components in docs/components.md. I think it would be more consistent to add to that, but maybe that's not discoverable enough. Either way this should probably live in the same place as that

Reply:
That's a good point. We could link to `doc/components` from here, potentially? The discoverability of this file is crucial I think - very easy to miss this info if it sits elsewhere.